chapter_linear-networks/linear-regression-concise.md

```diff
@@ -131,13 +148,19 @@ Gluon会自动推断每个层输入的形状。
 :begin_tab:`tensorflow`
 在Keras中，全连接层在`Dense`类中定义。
 由于我们只想得到一个标量输出，所以我们将该数字设置为1。
+:end_tab:
+
+:begin_tab:`paddle`
+在PaddlePaddle中，全连接层在`Linear`类中定义。
+值得注意的是，我们将两个参数传递到`nn.Linear`中。
+第一个指定输入特征形状，即2，第二个指定输出特征形状，输出特征形状为单个标量，因此为1。
+:end_tab:
```

Comment:
This block of paddle text should come after the paragraph below which is about tensorflow and keras and should be one whole text tab.